images/scripts/rhel.setup

```diff
@@ -208,6 +208,7 @@ tlog \
 valgrind \
 vdo \
 virt-install \
+wireguard-tools \
```

Comment:
This doesn't exist on rhel 8, so will fail. Keep it here, but remove it for centos-8-stream and rhel-8* in a rule similar to line 219.

Reply:
Done.

Question: The backslash `\` method of removing something out of a variable isn't part of bash, right? I couldn't reproduce it in a normal script.

Reply:
I'm not familiar with `\` being used to remove something from a variable. The forward-slash substitution is bash syntax. The backslash here is just to split a single logical line into multiple physical ones, to make them easier to patch and read.

Reply:
Sorry I meant forward slash :man_facepalming:. So it is a builtin bash feature(?) Nice!